### PR TITLE
Update define-command-argument-container.md

### PR DESCRIPTION
**Pull request by wangcanfengxs**

### CODE REVIEW

File: content/zh/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ content_template: templates/task
     kubectl get pods
     ```
 
-    查询结果显示在command-demo这个Pod下运行的容器已经启动完成
+   查询结果显示在command-demo这个Pod下运行的容器已经启动完成
```
Comment (xichengliudui) [inline content/zh/docs/tasks/inject-data-application/define-command-argument-container.md]:
```suggestion
   查询结果显示在 command-demo 这个 Pod 下运行的容器已经启动完成
```

**Comment by xichengliudui** [comment]
@wangcanfengxs  Thank you for your initial contribution. Please sign the cncf-cla agreement

File: content/zh/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ content_template: templates/task
...
 
 3. 如果要获取容器启动时执行命令的输出结果，可以通过Pod的日志进行查看
```
Comment (xichengliudui) [inline content/zh/docs/tasks/inject-data-application/define-command-argument-container.md]:
```suggestion
3. 如果要获取容器启动时执行命令的输出结果，可以通过 Pod 的日志进行查看
```

**Comment by xichengliudui** [comment]
ping

**Comment by tengqm** [comment]
Feel free to reopen when the CLA is signed. Thanks.

**Pull request closed without merging**